kratos/solving_strategies/strategies/explicit_solving_strategy_runge_kutta.h

```diff
@@ -388,7 +389,7 @@ class ExplicitSolvingStrategyRungeKutta : public ExplicitSolvingStrategy<TSparse
                 if (!it_dof->IsFixed()) {
                     const double mass = r_lumped_mass_vector(i_dof);
                     const auto k = row(rIntermediateStepResidualVectors, i_dof);
-                    r_u = r_u_old + (dt / mass) * std::inner_product(alphas.begin, alphas.end, k.begin(), 0.0);
+                    r_u = r_u_old + (dt / mass) * std::inner_product(alphas_begin, alphas_end, k.begin(), 0.0);
```

Comment:
The comment right below this (Github won't let me select it) is still true but no longer relevant, as we are now forced to use std::inner_product due to the types involved.

Reply:
Agree. However, as we're quite used to the boost one, I left it as a reasoning for the usage of the `std` one. Nonetheless, removing it would be fine for me if you consider it misleading.

Reply:
It's fine either way, let's leave it there.